src/components/Welcome.tsx

```diff
@@ -0,0 +1,16 @@
+const Welcome = () => {
+  return (
+    <div
+      id="Welcome"
+      className="gap-3 grid grid-cols-1 lg:grid-cols-4 pt-60 p-5 pb-20"
+    >
+      <div></div>
```

Comment:
Why the empty div? If this is intended, leave a comment inside the div, like this <div>{/*Don't remove this div*/}</div>